src/utils/helper.ts

```diff
@@ -13,5 +13,5 @@ export const isFunction = (v: any): v is Function => typeof v == 'function'
 export const mergeObjects = (a: any, b: any) => OBJECT.assign({}, a, b)
 
 const STR_UNDEFINED = 'undefined'
-export const hasWindow = typeof window != STR_UNDEFINED
-export const hasDocument = typeof document != STR_UNDEFINED
+export const hasWindow = () => typeof window != STR_UNDEFINED
```

Comment:
@huozhi Could you tell me why do we need to use a function here? I'm curious to learn.